modules/gapi/include/opencv2/gapi/video.hpp

```diff
@@ -169,6 +175,20 @@ calcOpticalFlowPyrLK(const GArray<GMat>    &prevPyr,
                            int              flags        = 0,
                            double           minEigThresh = 1e-4);
 
+/** @brief Gaussian Mixture based Background/Foreground Segmentation.
+The class generates a foreground mask.
+
+Output image is foreground mask, i.e. 8-bit unsigned 1-channel (binary) matrix @ref CV_8UC1.
+
+@note Functional textual ID is "org.opencv.videoanalysis.backgroundSubtractormog2"
+
+@param src input image: Floating point frame is used without scaling and should be in range [0,255].
+@ref CV_32FC3.
+
+@sa BackSubMOG2
+*/
+GAPI_EXPORTS GMat BackSubMOG2(const GMat& src);
```

Comment:
OpenCV implementation of this algorithm has several parameters. Why do we lose them?

```
    createBackgroundSubtractorMOG2(int history=500, double varThreshold=16,
                                   bool detectShadows=true);
```

---

> BackSubMOG2

Do we really want to obfuscate function name?

---

> `@sa BackSubMOG2`

Points on that? on this function again?

Reply:
Addressed